CHANGELOG.md

```diff
@@ -386,7 +386,7 @@ Unreleased` header.
 - Added `Window::set_transparent` to provide a hint about transparency of the window on Wayland and macOS.
 - On macOS, fix the mouse buttons other than left/right/middle being reported as middle.
 - On Wayland, support fractional scaling via the wp-fractional-scale protocol.
-- On web, fix removal of mouse event listeners from the global object upon window distruction.
+- On web, fix removal of mouse event listeners from the global object upon window destruction.
```

Comment:
```suggestion
- On Web, fix removal of mouse event listeners from the global object upon window destruction.
```
While we are at it.

Reply:
Applied that edit and one other instance of the same edit.